kentik_api_library/kentik_api/api_connection/api_connector.py

```diff
@@ -37,15 +55,29 @@ def send(self, api_call: APICall, payload: Optional[Dict[str, Any]] = None) -> A
             response = requests.delete(url, auth=self._auth, headers=self.DEFAULT_HEADERS, json=payload)
         else:
             raise ValueError(f"Improper API call method: {api_call.method}")
+        return response
 
-        self._validate_response(response)
+    def _get_api_query_url(self, url_path: str) -> str:
+        return self._api_url + url_path
 
-        return APICallResponse(response.status_code, response.text)
+    @staticmethod
+    def _handle_error(response: requests.Response) -> None:
```

Comment:
I suggest changing the name to check_for_error or even better raise_on_error; handle_error assumes that we are passing error as parameter

Reply:
Done